### DISCUSSION TITLE
[Bug] Model content field creation

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#395] [Help] [answered]
I detected a bug in creation of attributes and also in model content field creation. After successful creation of a field, I add directly another one. At saving it shows an error message: ID already exists. It seems it's changing the field added before to the new values. After a reload of the page, creation can be completed.

**Comment by sags95** [comment, marked as answer]
Hey @josias-hosang, have passed these details on to the team.

**Comment by steven-k1** [comment]
A fix has been applied and adding a content field should work as expected now
